plugins/main/bridge/bridge.go

```diff
@@ -55,6 +55,24 @@ type NetConf struct {
 	HairpinMode  bool   `json:"hairpinMode"`
 	PromiscMode  bool   `json:"promiscMode"`
 	Vlan         int    `json:"vlan"`
+	Mac          string `json:"mac,omitempty"`
```

Comment:
`Mac` field here is internal, it should be ignored when marshalling

Reply:
@EdDev this is the netconf for the overall Bridge, not the container interface. A 'mac' field here would be the MAC of the bridge itself, not the container veth. It shouldn't be here because per-container options are not specified int he top-level plugin config struct. That's what the runtime args and CNI_ARGS are for.

Reply:
Done